isis/src/base/apps/stretch/stretch.h

```diff
@@ -0,0 +1,19 @@
+#ifndef stretch_app_h
+#define stretch_app_h
+
+#include "TextFile.h"
+#include "Statistics.h"
+#include "ProcessByLine.h"
+#include "SpecialPixel.h"
+#include "Stretch.h"
+#include "PvlGroup.h"
+#include "PvlKeyword.h"
+#include "UserInterface.h"
```

Comment:
The only includes needed here are UserInterface, Pvl, and Cube. The rest should be moved to the implementation file.